cargo-crev/Cargo.toml

```diff
@@ -21,7 +21,7 @@ path = "src/lib.rs"
 
 [dependencies]
 atty = "0.2"
-cargo = "0.35"
+cargo = "0.38"
```

Comment:
There was some problem with `cargo` ...

Reply:
https://github.com/rust-lang/cargo/issues/7105

Reply:
Couple of calls changed args.

Reply:
Please just double check if it's OK.

Reply:
The call to acquire_package_cache_lock does the trick